docs/src/pages/guides/FHIRServerUsersGuide.md

```diff
@@ -2594,7 +2594,7 @@ For more information about topics related to configuring a FHIR server, see the
 
 - <b id="f5">5</b>
 
-    An external reference is a reference to a resource which is meaningful outside a particular request bundle.  The value typically includes the resource type and the resource identifier, and could  be an absolute or relative URL. Examples:  `https://fhirserver1:9443/fhir-server/api/v4/Patient/12345`, `Patient/12345`, etc. [↩](#a5)
+    An external reference is a reference to a resource which is meaningful outside a particular request bundle.  The value must include the resource type and the resource identifier, and could be an absolute or relative URL. Examples: `https://fhirserver1:9443/fhir-server/api/v4/Patient/12345`, `Patient/12345`, etc. [↩](#a5)
```

Comment:
Shouldn't the configuration setting be mentioned with this paragraph?

Reply:
Updated.